routes/course.js

```diff
@@ -0,0 +1,23 @@
+const router = require('express').Router()
+
+const idValidation = require('~/middlewares/idValidation')
+const asyncWrapper = require('~/middlewares/asyncWrapper')
+const { authMiddleware, restrictTo } = require('~/middlewares/auth')
+const isEntityValid = require('~/middlewares/entityValidation')
+
+const courseController = require('~/controllers/course')
+const Lesson = require('~/models/lesson')
+const {
+  roles: { TUTOR }
+} = require('~/consts/auth')
+
+const body = [{ model: Lesson, idName: 'lessons' }]
+
+router.use(authMiddleware)
+
+router.param('id', idValidation)
```

Comment:
i am not sure that here is need to use idValidation middleware for id param cause we don't endpoint with this param